Sound/Tidal/Pattern.hs

```diff
@@ -1457,9 +1457,8 @@ to change this use `toScale' size`.  Example:
 -}
 toScale' :: Int -> [Int] -> Pattern Int -> Pattern Int
 toScale' o s = fmap noteInScale
-  where note x = x `mod` length s
```

Comment:
Normally, the `octave` is not the length of the scale.

Reply:
True, but neither does the `octave` function – the `octave` function gets the octave to which the given input needs to be mapped.

Or do you mean that the argument `o` ought to be renamed?

Reply:
@telephon the `o` argument is meant to be "how many semitones before the scale repeats".  Or more literally "what offset should be used for note indices outside the given scale", e.g. if the scale is `[0, 4, 7]` what note should be returned for the 4th element.  That's not exactly what an "octave" is, but much of the time it's 12 anyway.

Reply:
yes, that's about what I would have expected. But that is not `length s`, is it?

Reply:
@telephon Just to clarify, are you referring to the argument `o` or the function `octave`? You're saying that one of those isn't the same as `length s`? Is it a question of nomenclature or correctness of implementation?

Reply:
It's fine, sorry - I misread the function implementation (I should have looked at the whole file, not just the changes).